warrior/Framework/Utils/email_utils.py

```diff
@@ -9,20 +9,19 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-'''
+"""
 # Utility to send email using smtp
 # Import smtplib for the actual sending function
-
 import smtplib
 import os
 from os.path import basename
 from email import encoders
-from email.MIMEBase import MIMEBase
+from email.mime.base import MIMEBase
```

Comment:
Changing the module to mime.base as as the modules have been renamed.
Reference: https://docs.python.org/2/library/email.html